### PR TITLE
Doc corrections for resource group to intimate id and subnet floating ip changes 

### PR DESCRIPTION
**Pull request by BShrivastav**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: website/docs/d/is_backup_policies.html.markdown [hunk 1]
```diff
@@ -63,7 +63,7 @@ In addition to all argument reference list, you can access the following attribu
       - `id` - (String) The unique identifier for this backup policy plan.
       - `name` - (String) The unique user-defined name for this backup policy plan.
       - `resource_type` - (String) The type of resource referenced.
-    - `resource_group` - (List) The resource group for this backup policy. 
+    - `resource_group` - (List) The Resource group, for this backup policy.
```
Comment (ujjwal-ibm) [inline website/docs/d/is_backup_policies.html.markdown:66]: *resource group object

File: website/docs/d/is_ipsec_policies.html.markdown [hunk 2]
```diff
@@ -42,7 +42,7 @@ In addition to all argument references listed, you can access the following attr
 	- `key_lifetime` - (Required, Integer) The key lifetime in seconds.
 	- `name` - (Required, String) The user-defined name for this IPsec policy.
 	- `pfs` - (Required, String) Perfect Forward Secrecy.
-	- `resource_group` - (Required, List) The resource group for this IPsec policy.
+	- `resource_group` - (Required, List) The resource group id, for this IPsec policy.
```
Comment (ujjwal-ibm) [inline website/docs/d/is_ipsec_policies.html.markdown:45]: object

File: website/docs/d/is_ipsec_policy.html.markdown [hunk 3]
```diff
@@ -68,7 +68,7 @@ In addition to all argument references listed, you can access the following attr
 - `key_lifetime` - (Integer) The key lifetime in seconds.
 - `name` - (String) The user-defined name for this IPsec policy.
 - `pfs` - (String) Perfect Forward Secrecy.
-- `resource_group` - (List) The resource group for this IPsec policy.
+- `resource_group` - (List) The resource group id, for this IPsec policy.
```
Comment (ujjwal-ibm) [inline website/docs/d/is_ipsec_policy.html.markdown:71]: object

**Review by ujjwal-ibm — CHANGES_REQUESTED** [review]

File: website/docs/d/is_ipsec_policies.html.markdown (hunk 2 above)
Comment (ujjwal-ibm) [reply website/docs/d/is_ipsec_policies.html.markdown:45]: remove required from all attributes

File: website/docs/d/is_backup_policies.html.markdown (hunk 1 above)
Comment (BShrivastav) [reply website/docs/d/is_backup_policies.html.markdown]: done.

**Comment by BShrivastav** [comment]
Raised Upstream - https://github.com/IBM-Cloud/terraform-provider-ibm/pull/4025

**Pull request closed without merging**